.claude/scripts/fetch_github_pr.py

```
@@ -0,0 +1,346 @@
+            response = requests.get(url, headers=headers, timeout=30)
+            response.raise_for_status()
+            return response.json()
+        raise
+
+
+def fetch_pr_comments(repo: str, pr_number: int, github_token: Optional[str] = None) -> List[Dict]:
+    """Fetch PR comments from GitHub API"""
+    url = f"https://api.github.com/repos/{repo}/issues/{pr_number}/comments"
+    headers = {"Accept": "application/vnd.github.v3+json"}
+
+    if github_token and github_token != 'your_github_token_here':
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to: "github_token"

(S105)